bin/varnishtest/tests/c00113.vtc

```diff
@@ -0,0 +1,31 @@
+varnishtest "probe expect close"
+
+server s0 {
+	rxreq
+	txresp
+	expect_close
+} -dispatch
+
+varnish v1 -vcl+backend {
+	probe default {
+		.window = 1;
+		.threshold = 1;
+		.timeout = 1s;
+		.interval = 0.1s;
+	}
```

Comment:
Please add an explicit `.expect_close = true;` for coverage, we already have test cases without this field.